crates/build/re_types_builder/src/codegen/rust/api.rs

```diff
@@ -1591,6 +1645,25 @@ fn quote_builder_from_obj(reporter: &Reporter, objects: &Objects, obj: &Object)
             }
         });
 
+        let quoted_eager_required = required.iter().map(|field| {
+            let field_name = format_ident!("{}", field.name);
+            let descr_fn_name = format_ident!("descriptor_{field_name}");
+
+            let (_, unwrapped) = quote_field_type_from_typ(&field.typ, true);
+            if unwrapped {
+                // This was originally a vec/array!
```

Comment:
This comment confused me for a bit. What you seem to mean is that this component is defines as being a many-component (as opposed to being a mono-component)?

Reply:
same below

Reply:
That's copy pasta from the existing code but yes that's what it means